No overload matches error when trying to run locally

**Summary**
As I tried to run stryker locally I ran into this error.
I did 'npm install' and then 'npm run build' and got this issue.

```
npm run build

> prebuild
> npm run generate


> generate
> node tasks/generate-json-schema-to-ts.js && node tasks/generate-mono-schema.js

✅ packages/jest-runner/schema/jest-runner-options.json -> packages/jest-runner/src-generated/jest-runner-options.ts
✅ packages/karma-runner/schema/karma-runner-options.json -> packages/karma-runner/src-generated/karma-runner-options.ts
✅ packages/mocha-runner/schema/mocha-runner-options.json -> packages/mocha-runner/src-generated/mocha-runner-options.ts
✅ packages/api/schema/stryker-core.json -> packages/api/src-generated/stryker-core.ts
✅ packages/jasmine-runner/schema/jasmine-runner-options.json -> packages/jasmine-runner/src-generated/jasmine-runner-options.ts
✅ packages/cucumber-runner/schema/cucumber-runner-options.json -> packages/cucumber-runner/src-generated/cucumber-runner-options.ts
✅ Merged 6 schemas into packages/core/schema/stryker-schema.json

> build
> tsc -b && lerna run build

packages/jest-runner/test/unit/jest-plugins/jest-environment-generic.spec.ts:16:37 - error TS2769: No overload matches this call.
  Overload 1 of 2, '(event: AsyncEvent, state: State): void | Promise<void> | undefined', gave the following error.
    Argument of type 'import("/Users/mac/Documents/stryker-js/packages/jest-runner/node_modules/@jest/types/build/Circus").AsyncEvent' is not assignable to parameter of type 'import("/Users/mac/Documents/stryker-js/packages/jest-runner/node_modules/@jest/console/node_modules/@jest/types/build/Circus").AsyncEvent'.
      Type '{ name: "setup"; testNamePattern?: string | undefined; runtimeGlobals: JestGlobals; parentProcess: Process; }' is not assignable to type 'AsyncEvent'.
        Type '{ name: "setup"; testNamePattern?: string | undefined; runtimeGlobals: JestGlobals; parentProcess: NodeJS.Process; }' is not assignable to type '{ name: "setup"; testNamePattern?: string | undefined; runtimeGlobals: JestGlobals; parentProcess: NodeJS.Process; }'. Two different types with this name exist, but they are unrelated.
          The types of 'runtimeGlobals.it.concurrent.only.each' are incompatible between these types.
            Type 'Each<import("/Users/mac/Documents/stryker-js/packages/jest-runner/node_modules/@jest/types/build/Global").ConcurrentTestFn>' is not assignable to type 'Each<import("/Users/mac/Documents/stryker-js/packages/jest-runner/node_modules/@jest/console/node_modules/@jest/types/build/Global").ConcurrentTestFn>'.
              Type '(table: EachTable, ...taggedTemplateData: unknown[]) => (title: string, test: EachTestFn<ConcurrentTestFn>, timeout?: number | undefined) => void' is not assignable to type 'Each<ConcurrentTestFn>'.
                Type '(table: EachTable, ...taggedTemplateData: unknown[]) => (title: string, test: EachTestFn<ConcurrentTestFn>, timeout?: number | undefined) => void' is not assignable to type '() => () => void'.
  Overload 2 of 2, '(event: SyncEvent, state: State): void | undefined', gave the following error.
    Argument of type 'AsyncEvent' is not assignable to parameter of type 'SyncEvent'.
      Type '{ name: "setup"; testNamePattern?: string | undefined; runtimeGlobals: JestGlobals; parentProcess: Process; }' is not assignable to type 'SyncEvent'.
        Property 'error' is missing in type '{ name: "setup"; testNamePattern?: string | undefined; runtimeGlobals: JestGlobals; parentProcess: Process; }' but required in type '{ name: "error"; error: any; }'.

16     await jestEnv.handleTestEvent?.(producers.createCircusRunStartEvent(), producers.createCircusState());
                                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  packages/jest-runner/node_modules/@jest/console/node_modules/@jest/types/build/Circus.d.ts:64:5
    64     error: Exception;
           ~~~~~
    'error' is declared here.


Found 1 error.
```

**Stryker config**


```json
{
}
```

**Test runner config**

<!--- Please Put your Jest / Karma / Jasmine / Mocha (...) configuration here. -->

**Stryker environment**

<!-- Please list your stryker plugins + versions here (output of `npm ls | grep stryker` or `npm ls | findstr "stryker"` on Windows) -->

```
+-- @stryker-mutator/core@x.x.x
+-- @stryker-mutator/xxx-xxx@x.x.x
```

<!-- Please also add the test runner you are using.  Examples:-->

```
+-- mocha@x.x.x
+-- jest@x.x.x
+-- karma@x.x.x
+-- angular-cli@x.x.x
+-- react-scripts@x.x.x
```

**Test runner environment**

<!-- Please add your test command here (probably the command you use in `npm test`) command -->

```shell
# Test command
```

<!-- Please add any config files that are used by your test runner. For example jest.config.js, karma.conf.js, angular.json, mocha.opts, ...  -->

```json
```


**Your Environment**

| software         | version(s)
| ---------------- | -------
| node             | v16.13.1
| npm              | 8.1.2
| Operating System | macOS Monterey 12.1

**Add stryker.log**

<!-- Please add your stryker.log file. This file can be generated using `stryker run --fileLogLevel trace`. You can drag and drop it here. -->
<!-- Your source code is never logged to this file, however file names are logged. Feel free to obfuscate those log messages if you think it is a problem -->

--- a/packages/jest-runner/package.json
+++ b/packages/jest-runner/package.json
@@ -41,14 +41,14 @@
   },
   "homepage": "https://stryker-mutator.io/docs/stryker-js/jest-runner",
   "devDependencies": {
-    "@jest/types": "~27.4.2",
+    "@jest/types": "~27.5.0",
     "@stryker-mutator/test-helpers": "5.6.1",
-    "@testing-library/jest-dom": "^5.12.0",
-    "@testing-library/react": "^12.0.0",
-    "@types/node": "^17.0.0",
-    "@types/semver": "~7.3.6",
-    "jest": "~27.4.0",
-    "jest-circus": "~27.4.1",
+    "@testing-library/jest-dom": "^5.16.2",
+    "@testing-library/react": "^12.1.2",
+    "@types/node": "^17.0.15",
+    "@types/semver": "~7.3.9",
+    "jest": "~27.5.0",
+    "jest-circus": "~27.5.0",
     "jest-environment-jsdom-sixteen": "^2.0.0"
   },
   "peerDependencies": {
@@ -59,7 +59,7 @@
     "@stryker-mutator/api": "5.6.1",
     "@stryker-mutator/util": "5.6.1",
     "semver": "~7.3.5",
-    "tslib": "~2.3.0"
+    "tslib": "~2.3.1"
   },
   "initStrykerConfig": {
     "coverageAnalysis": "perTest"

